Nos/Service/NamesAPI.swift

```diff
@@ -51,15 +51,33 @@ class NamesAPI {
         throw Error.unexpected
     }
 
+    /// Verifies that a given username is free to claim in nos.social
     func verify(username: String, keyPair: KeyPair) async throws -> Bool {
```

Comment:
The comment explains it but I wish the function signature was more descriptive about what was being verified. Reading the call site makes it sound like we are verifying that username belongs to this keypair or something, but actually we are checking whether or not the given username is free (do we even need a keypair for this stage?). Maybe I am confusing things.

Reply:
Yeah, you're right, let's think a different name